website/content/en/configurations/identity-access-modes/service-principal-mode.md

```diff
@@ -4,7 +4,7 @@ title: "Service Principal"
 linkTitle: "Service Principal"
 weight: 1
 description: >
-  Use a Service Principal to access Keyvault.
+  Use a Service Principal to access Keyvault. ** This is currently the only way to connect to Azure Key Vault from a non Azure environment.

```

Comment:
Could you please add this as a note? For eg refer below line stating `Supported with Linux and Windows`.

You can check website preview [here](https://deploy-preview-534--secrets-store-csi-driver-provider-azure.netlify.app/)